tests/unittests/CoreGraphics.Drawing/CGContextDrawing_FillTests.cpp

```diff
@@ -0,0 +1,105 @@
+//******************************************************************************
```

Comment:
This file has a _lot_ of duplicated code. Can you refactor it into reusable functions or parameterized tests?